c/driver/postgresql/connection.cc

```diff
@@ -146,15 +185,17 @@ class PqGetObjectsHelper {
   }
 
   AdbcStatusCode GetObjects() {
-    PqResultHelper curr_db_helper = PqResultHelper{conn_, "SELECT current_database()"};
-    if (curr_db_helper.Status() == PGRES_TUPLES_OK) {
-      assert(curr_db_helper.NumRows() == 1);
-      auto curr_iter = curr_db_helper.begin();
-      PqResultRow db_row = *curr_iter;
-      current_db_ = std::string(db_row[0].data);
-    } else {
-      return ADBC_STATUS_INTERNAL;
-    }
+    std::vector<std::string> _;
```

Comment:
This is pointless to add an empty vector. I'm guessing we should just add an alternate constructor, but could also go in a separate class since the Prepare/Execute interaction is overkill for something without parameters

Reply:
You can write this as `PqResultHelper helper(conn_, "QUERY", {}, error_};`

Reply:
Though yeah, just add a second constructor for `PqResultHelper`.

Reply:
OK added the alternate constructor. The caller is still responsible for calling `Prepare` in this case even though it is overkill when there are no parameters. Could either create a separate class for statements that don't need to be prepared, or make the class more intelligent to recognize if it needs it or not. Leaving that as a follow up exercise